packages/orm/src/EntityManager.ts

```diff
@@ -862,7 +891,7 @@ async function addReactiveAsyncDerivedValues(todos: Record<string, Todo>): Promi
       (await followReverseHint([...todo.inserts, ...todo.updates], reverseHint)).forEach((entity) => {
         const todo = getTodo(todos, entity);
         if (!todo.inserts.includes(entity) && !todo.updates.includes(entity) && !entity.isDeletedEntity) {
-          todo.updates.push(entity);
+          todo.updates.push(util.types.isProxy(entity) ? entity : proxyEntity(entity, getEm(entity)["flushSecret"]));
```

Comment:
Do we need to de-proxy this for the `.includes` checked right before this line?

Er, wait, you're actually creating a proxy...can we keep these as not-proxies and then just create the proxy in `runHook`?

Reply:
somehow non-proxied entities were sneaking in here.  This was causing the code to blow up later when trying to fetch some meta data that proxies are bad at.  This was a quick and dirty fix, but should probably figure out a better one.